apps/cookbook/src/app/showcase/badge-showcase/badge-showcase.component.html

```diff
@@ -1,8 +1,67 @@
-<div class="example">
-  <cookbook-badge-example></cookbook-badge-example>
-  <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
-  <h4>Properties:</h4>
-  <cookbook-api-description-properties
-    [properties]="properties"
-  ></cookbook-api-description-properties>
-</div>
+<p>
+  Badges are inlines block elements that often are used on top of or close to another component.
+  They can be used with numbers, text and icons. A small badge version also exists which does not
+  support slotted content but can be used to indicate status.
+</p>
+
+<kirby-divider [hasMargin]="true"></kirby-divider>
+
+<h2>Examples</h2>
+<h3>Badge with numbers</h3>
+<p>
+  Using a badge with an number can be a good way to indicate status on an element.
```

Comment:
```suggestion
  Using a badge with a number can be a good way to indicate specific status on an element.
```
Consider this wording to mimic the zeplin guideline

Reply:
I've added back in 'specific'. It just seems kinda vague to me. But I guess that is perhaps something that should be taken up with UX.